core/src/main/java/io/grpc/internal/HedgingPolicy.java

```diff
@@ -83,6 +81,7 @@ public String toString() {
     /**
      * This method is used no more than once for each call. Never returns null.
```

Comment:
Optional since I guess this whole block is marked for deletion, but "Never returns null" contradicts the new annotation.